module/VuFind/src/VuFind/View/Helper/Root/SearchTabs.php

```diff
@@ -331,90 +303,41 @@ protected function remapBasicSearch(
             . $results->getUrlQuery()->getParams(false);
     }
 
-    /**
-     * Create information representing a basic search tab.
-     *
-     * @param string $id             Tab ID
-     * @param string $class          Search class ID
-     * @param string $label          Display text for tab
-     * @param string $newUrl         Target search URL
-     * @param string $permissionName Name of a permissionrule
-     *
-     * @return array
-     */
-    protected function createBasicTab($id, $class, $label, $newUrl, $permissionName)
-    {
-        return [
-            'id' => $id,
-            'class' => $class,
-            'label' => $label,
-            'permission' => $permissionName,
-            'selected' => false,
-            'url' => $newUrl,
-        ];
-    }
-
     /**
      * Create information representing a tab linking to "search home."
      *
-     * @param string $id             Tab ID
-     * @param string $class          Search class ID
-     * @param string $label          Display text for tab
-     * @param array  $filters        Tab filters
-     * @param string $permissionName Name of a permissionrule
+     * @param string $class   Search class ID
+     * @param array  $filters Tab filters
      *
-     * @return array
+     * @return string
      */
-    protected function createHomeTab($id, $class, $label, $filters, $permissionName)
+    protected function getHomeTabUrl($class, $filters)
```

Comment:
I realize that this is a pre-existing problem, but the comment at the top of this method is duplicated in getAdvancedTabUrl(), and I'm not sure if it really makes sense in either context.

Reply:
I guess that's my fault that I didn't update the comments. Done.